### ISSUE TITLE
Having Trouble downloading extracting files from BLACK v3.5.0(complete) without the usage of WinZip (attempting to use 7zip instead)

### ISSUE THREAD

**Issue by FadelusN100** [skotlex/ffxiv-material-ui#251]
Is this file compatible with 7zip, if so could you tell me how to extract it properly, if not are there any other free softwares i could use to extract the file

**Comment by sunxyw** [comment]
It should be compatible with 7zip, I always use 7zip to extract the files. For me, just double click the zip file and drag the files out.

**Comment by AmenneHolelane** [comment]
Correct, any tool that can extract a .zip file will work with the .zip released file

**Comment by AmenneHolelane** [comment]
> It should be compatible with 7zip, I always use 7zip to extract the files. For me, just double-click the zip file and drag the files out.
> 
> 
> 
> Gif here:
> 
> ![圖片](https://i.imgur.com/w8bNN79.gif)
> 
> 
> 
> 

You can also just right click > extract as I think it adds itself to the menu by default

**Comment by FadelusN100** [comment]
It worked out perfectly. tysm
